### PR TITLE
14 receive multiple requests

### LINKED ISSUE #14: Receive multiple booking requests
```
As a user
So that I can entertain multiple offers
I want to be able to receive multiple booking requests until one is confirmed
```

### PR DESCRIPTION
**Pull request by jaxdid**
closes #14

### CODE REVIEW

File: app/app.rb [hunk 1]
```diff
@@ -69,7 +69,14 @@ def validate_dates(space)
...
+      if current_user.id == space.user_id
+        redirect('/spaces')
+      end
+    end
+
     def validate(space)
+      validate_space(space)
       validate_dates(space)
       booking_from = space.bookings.map(&:from_date)
       booking_to = space.bookings.map(&:end_date)
```
Comment (russellvaughan) [inline app/app.rb]: just a little thing but is it maybe worth renaming one of this to something a  little bit different?  Validate_space(space) and validate(space) are a little bit hard to follow?

**Comment by RPiper93** [comment]
:shipit:
